config/config.py

```diff
@@ -528,18 +626,34 @@ def build_config(
     #########################
     # TES Shifts
     #########################
+    # configuration.add_shift(
+    #     SystematicShift(
+    #         name="tauES_1prong0pizeroDown",
+    #         shift_config={"global": {"tau_ES_shift_DM0": 0.998}},
+    #         producers={"global": taus.TauPtCorrection},
+    #         ignore_producers={"mt": [pairselection.LVMu1, muons.VetoMuons]},
+    #     )
+    # )
+    # configuration.add_shift(
+    #     SystematicShift(
+    #         name="tauES_1prong0pizeroUp",
+    #         shift_config={"global": {"tau_ES_shift_DM0": 1.002}},
+    #         producers={"global": taus.TauPtCorrection},
+    #         ignore_producers={"mt": [pairselection.LVMu1, muons.VetoMuons]},
+    #     )
+    # )
     configuration.add_shift(
         SystematicShift(
-            name="tauES_1prong0pizeroDown",
-            shift_config={"global": {"tau_ES_shift_DM0": 0.998}},
+            name="tauES_Down",
```

Comment:
I think i mentioned this wrong in the past, so we use the tauES splitted by decay mode for `et`, `mt`, `tt`, for TauID, we have a difference in the uncertainty scheme, where in tt we split by decay mode and in `et` and `mt` we split by pt. 

sorry ..

Reply:
Ok, I will split the tauES uncertainty in the decay modes. But I will do another issue for that.

Reply:
perfect !

Reply:
I made an issue for that https://github.com/KIT-CMS/CROWN/issues/136